lib/pubspec.lock

```diff
@@ -0,0 +1,167 @@
+# Generated by pub
```

Comment:
Could you delete `lib/pubspec.yaml` and `lib/pubspec.lock`? The pubspec files should only be in the top-level directory.